server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -2048,7 +2048,7 @@ private List<File> buildManifests(String platform) throws ExtenderException {
         return out;
     }
 
-    private List<File> writeLog() {
+    List<File> writeLogs() {
```

Comment:
Why the rename, it has one log?

The "list" was only for the "addAll" below.
I suggest changing that to an "add(log)" instead.

Reply:
I was considering this as well. I'll change it now that you also point it out!